internal/rest/avi_obj_vs.go

```diff
@@ -255,6 +256,11 @@ func (rest *RestOperations) AviVsSniBuild(vs_meta *nodes.AviVsNode, rest_method
 
 		var httpPolicyCollection []*avimodels.HTTPPolicies
 		internalPolicyIndexBuffer := int32(11)
+
+		sort.Slice(vs_meta.HttpPolicyRefs, func(i, j int) bool {
```

Comment:
For this type of sorting, we aren't considering the noPath condition. but with EVH we are doing that. If you had a host without any path - will path[0] have a "/"?

Reply:
@sudswasavi : Thanks. Yes. I missed that. With latest changes fixed ordering of no path and with path httpps in VS